tachyon/crypto/hashes/sponge/poseidon2/poseidon2_external_matrix.h

```diff
@@ -49,8 +49,8 @@ class Poseidon2ExternalMatrix {
       Derived::DoApply(block);
     }
 
-    math::Vector<Field> v_tmp(4);
-    v_tmp << Field::Zero(), Field::Zero(), Field::Zero(), Field::Zero();
+    std::array<Field, 4> v_tmp = {Field::Zero(), Field::Zero(), Field::Zero(),
+                                  Field::Zero()};
```

Comment:
Shouldn't the function signature change as well?

`static void Apply(math::Vector<Field>& v)`